common/appfile.go

```diff
@@ -23,10 +23,10 @@ var (
 
 // AppFile defines the internal structure of a app.yaml/json/yml
 type AppFile struct {
```

Comment:
that's happening again, can we reuse Fn models in the CLI because we had to manually copy-paste from server code?

Reply:
the fn models don't generate the yaml tags at the moment :/